app/controllers/lookups_controller.rb

```diff
@@ -1,7 +1,12 @@
 class LookupsController < ApplicationController
   def by_base_path
+    # return content_ids for content that is visible on the live site
+    # withdrawn items are still visible
+    states = %w(published unpublished)
```

Comment:
@MatMoore I think this does return a base_path iff the item has ever been published. There are 4 possible states, draft, published, superseded, unpublished. Items in draft need not ever have been published. Items in superseded necessarily have a published successor, which may become unpublished, but is never deleted or changed to any other state. So the filter to include published and unpublished should be sufficient to find the base_path for all content that has ever been published.